src/boot/java/io/github/brassmc/brassloader/boot/discovery/ModDiscovery.java

```diff
@@ -35,6 +35,10 @@ public List<Resource> completeScan(IModuleLayerManager layerManager) {
         try {
             final List<SecureJar> mods = new ArrayList<>();
 
+            if(Files.notExists(MODS_FOLDER)) {
+                Files.createDirectories(MODS_FOLDER);
+            }
```

Comment:
```suggestion
            Files.createDirectories(MODS_FOLDER);
```
Unnecessary `Files.exists` check, the native impl of `Files.createDirectories` does this already.